sdks/python/apache_beam/portability/api/__init__.py

```diff
@@ -1,21 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-"""For internal use only; no backwards-compatibility guarantees.
-
-Automatically generated when running setup.py sdist or build[_py].
```

Comment:
This file removed in favor of a real autogenerated file. The new file looks something like:

```python
from .org.apache.beam.model import pipeline
from .org.apache.beam.model import jobmanagement
from .org.apache.beam.model import fnexecution
from .org.apache.beam.model import interactive

external_transforms_pb2 = pipeline.external_transforms_pb2
metrics_pb2_grpc = pipeline.metrics_pb2_grpc
schema_pb2_grpc = pipeline.schema_pb2_grpc
beam_runner_api_pb2_urns = pipeline.beam_runner_api_pb2_urns
schema_pb2 = pipeline.schema_pb2
metrics_pb2 = pipeline.metrics_pb2
external_transforms_pb2_grpc = pipeline.external_transforms_pb2_grpc
standard_window_fns_pb2_urns = pipeline.standard_window_fns_pb2_urns
standard_window_fns_pb2_grpc = pipeline.standard_window_fns_pb2_grpc
endpoints_pb2 = pipeline.endpoints_pb2
standard_window_fns_pb2 = pipeline.standard_window_fns_pb2
external_transforms_pb2_urns = pipeline.external_transforms_pb2_urns
metrics_pb2_urns = pipeline.metrics_pb2_urns
beam_runner_api_pb2 = pipeline.beam_runner_api_pb2
endpoints_pb2_grpc = pipeline.endpoints_pb2_grpc
beam_runner_api_pb2_grpc = pipeline.beam_runner_api_pb2_grpc
beam_artifact_api_pb2_urns = jobmanagement.beam_artifact_api_pb2_urns
beam_expansion_api_pb2_grpc = jobmanagement.beam_expansion_api_pb2_grpc
beam_artifact_api_pb2 = jobmanagement.beam_artifact_api_pb2
beam_artifact_api_pb2_grpc = jobmanagement.beam_artifact_api_pb2_grpc
beam_job_api_pb2 = jobmanagement.beam_job_api_pb2
beam_job_api_pb2_grpc = jobmanagement.beam_job_api_pb2_grpc
beam_expansion_api_pb2 = jobmanagement.beam_expansion_api_pb2
beam_fn_api_pb2_grpc = fnexecution.beam_fn_api_pb2_grpc
beam_provision_api_pb2 = fnexecution.beam_provision_api_pb2
beam_provision_api_pb2_grpc = fnexecution.beam_provision_api_pb2_grpc
beam_fn_api_pb2 = fnexecution.beam_fn_api_pb2
beam_interactive_api_pb2_grpc = interactive.beam_interactive_api_pb2_grpc
beam_interactive_api_pb2 = interactive.beam_interactive_api_pb2
```

This allows imports of the form:
```python
from apache_beam.portability.api import beam_interactive_api_pb2
```

without the rest of the package needing to know the shape of the generated bindings

Reply:
@tvalentyn any opinions on the package structure change with regards to how imports are now done?

Reply:
I'm fine with this given the constraints.